src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux/src/main/scala/server/Server.scala

```diff
@@ -20,7 +20,7 @@ object Server {
       new File("/etc/ssl/server.key"),
     )
     val sslConfig = SslServerConfiguration(
-      keyCredentials,
+      keyCredentials=keyCredentials,
```

Comment:
The order of these positional arguments changed. Switching to named parameters should hopefully future proof this.